CMakeLists.txt

```diff
@@ -226,22 +228,22 @@ IF(${USE_FFTW})
         FIND_LIBRARY(FFTW3 libfftw3.a HINTS ${FFTW_LIB})
         FIND_LIBRARY(FFTW3_MPI libfftw3_mpi.a HINTS ${FFTW_LIB})
     ENDIF()
-    MESSAGE(${FFTW3})
-    MESSAGE(${FFTW3_MPI})
+    MESSAGE(STATUS "Found FFTW lib at ${FFTW3}")
+    MESSAGE(STATUS "Found FFTW mpi lib at ${FFTW3_MPI}")
     TARGET_LINK_FFTW(exadg)
 ENDIF()
 
 # LIKWID
-FIND_LIBRARY(LIKWID likwid HINTS ${LIKWID_LIB})
-MESSAGE(${LIKWID}) 
-IF(LIKWID)
+OPTION(EXADG_WITH_LIKDWID "Use LIKWID" OFF)
+IF(${EXADG_WITH_LIKWID})
+    FIND_LIBRARY(LIKWID likwid HINTS ${LIKWID_LIB})
+    MESSAGE(STATUS "Found LIKWID lib at ${LIKWID}")
     TARGET_COMPILE_DEFINITIONS(exadg PUBLIC LIKWID_PERFMON)
```

Comment:
@kronbichler When replacing `LIKWID_PERFMON` by `EXADG_WITH_LIKWID` in this line and the rest of the code, this leads to the error

Marker API result file does not exist. This may happen if the application has not called LIKWID_MARKER_CLOSE.

Why? I thought we do the same with `EXADG_WITH_FFTW`, `DEAL_II_WITH_TRILINOS`, etc.? Where is the difference?

Reply:
The macro `LIKWID_PERFMON` is used inside Likwid, i.e., we need to pass that precise macro into the Likwid machinery for compilation.